include/alpaka/onAcc/Warp.hpp

```diff
@@ -0,0 +1,273 @@
+/* Copyright 2025 Mehmet Yusufoglu, René Widera
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * Bridges runtime accelerator instances to the trait-based warp intrinsics so kernels can call them without tags.
+ * Exposes device-safe `alpaka::onAcc::warp::*` wrappers for ballots, shuffles, and lane queries.
+ * Reuses the compile-time warp trait specialisations instead of duplicating backend-specific logic in kernels.
+ * Supplies a uniform warp API across CUDA, HIP, SYCL, and host-emulation accelerators.
+ *
+ * Some example usages:
+ * - consteval `alpaka::getWarpSize(api::Cuda{}, deviceKind::NvidiaGpu{})` for tag-driven compile-time logic.
+ * - runtime `alpaka::getWarpSize(deviceSpec)` when host code needs the width before launching work.
+ * - device-side `alpaka::onAcc::warp::getSize(acc)` to query the active warp inside a kernel.
+ * - device-side `alpaka::onAcc::warp::shfl(acc, 42, 0u) == 42`
+ */
+
+#pragma once
```

Comment:
Please rename this file to `warp.hpp`. We only name files with a capital letter in cases where it contains a class with the same name. This is a collection of interface functions there we start the file with a small letter.